conjur/cli.py

```diff
@@ -185,7 +185,7 @@ def handle_hostfactory_logic(cls, args: list = None, client=None):
             hostfactory_logic = HostFactoryLogic(client)
 
             days = args.duration_days if args.duration_days else 0
-            hours = args.duration_hours if args.duration_hours else 0
+            hours = args.duration_hours if args.duration_hours else 1
```

Comment:
i'm guessing this is approved by Sapir?

Reply:
set this back to 0

Reply:
what is the requirement from product?

Reply:
we default to 1 (see create_token_data)
I moved this back to 0 because defaults shouldnt be set in the view